homeassistant/components/sonos/media_browser.py

```diff
@@ -265,41 +266,41 @@ async def root_payload(
             )
         )
 
-    if "plex" in hass.config.components:
+    if await hass.async_add_executor_job(
+        partial(media.library.browse_by_idstring, "tracks", "", max_items=1)
+    ):
         children.append(
             BrowseMedia(
-                title="Plex",
+                title="Music Library",
                 media_class=MEDIA_CLASS_DIRECTORY,
```

Comment:
```suggestion
                media_class=MEDIA_CLASS_APP,
```

Reply:
that's not an app, that is the music library of local media in sonos.